catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -435,18 +444,21 @@ protected void showMergeDialog(List<T> selectedItems) {
 			ToastUtil.showError(getContext(), R.string.am_merge_error);
 		} else {
 			TextInputDialog.Builder builder = new TextInputDialog.Builder(getContext());
-
 			builder.setHint(getString(R.string.project_name_label))
 					.setPositiveButton(getString(R.string.ok), (TextInputDialog.OnClickListener) (dialog, textInput)
 							-> {
-						mergeProjects(selectedItems, textInput);
+						try {
+							mergeProjects(selectedItems, textInput);
```

Comment:
error handling and information for the user should be handeled here.
(the toasts in the method are only visible if no exceptions are thrown, but if it ends up with a null project)